.github/workflows/docker-publish.yml

```diff
@@ -29,15 +29,40 @@ jobs:
           username: ${{ env.DOCKERHUB_USERNAME }}
           password: ${{ env.DOCKERHUB_TOKEN }}
 
-      - name: Build and push Docker image
+      - name: Build and push Docker image (Development)
+        if: github.event_name == 'push'
         uses: docker/build-push-action@v4
         with:
           context: .
+          file: Dockerfile.dev
```

Comment:
I mean to ask if there is a reason can't use the main docker file. This will be for use in k8s right?